test/document_stores/test_faiss.py

```diff
@@ -1,18 +1,11 @@
-import sys
-import os
-
-import yaml
```

Comment:
I took the chance to remove some unused imports in this module, not related to the PR main goal